### PR TITLE
[ Fix ] Fix android build error with meson @open sesame 07/26 12:11

### PR DESCRIPTION
**Pull request by jijoongmoon**
This commit includes,
  . Fix android build issues
    : openblas link and download

**Self evaluation:**
1. Build test:	 [X]Passed [ ]Failed [ ]Skipped
2. Run test:	 [X]Passed [ ]Failed [ ]Skipped

Signed-off-by: jijoong.moon <jijoong.moon@samsung.com>

### CODE REVIEW

File: jni/Android.mk [hunk 1]
```diff
@@ -90,13 +90,13 @@ $(info $(shell $(NNTRAINER_JNI_ROOT)/prepare_openblas.sh $(NDK_LIBS_OUT)))
 endif #MAKECMDGOALS
 endif #OPENBLAS_ROOT
 
+include $(CLEAR_VARS)
```
Comment (kparichay) [inline jni/Android.mk:93]: maybe this should go between L100 and L101?

Comment (jijoongmoon) [reply jni/Android.mk:93]: right. 102 is better. fixed.

**Review by kparichay — APPROVED** [review]
LGTM

**Review by zhoonit — APPROVED** [review]
LGTM

**Pull request merged**